frontend/src/hooks/useFormData/initialFormData.tsx

```diff
@@ -1,41 +1,30 @@
-import { ErrorMessage } from "../../constants";
-import { FieldError, FormData } from "../../types";
+import { FormData } from "../../types";
 
 const initialFormData: FormData = {
     winner: {
         value: null,
         error: null,
-        validate: function _() {
-            return detectMissingInput(this.value);
-        },
+        validate: alwaysValid,
```

Comment:
I moved responsibility for checking required fields into the `validateField` and `validateForm` helpers, because null/non-null validation is specifically neeeded to ensure the type coercion from `FormData` to `ValidFormData` isn't a bad idea, so I wanted all the logic to at least be closer together.

So all the `validate` functions on individual fields are placeholders now. If they end up staying that way (haven't thought about best place for dependent field validation yet), will wipe them out!